libs/components/core/src/lib/modules/live-announcer/fixtures/live-announcer.component.fixture.ts

```diff
@@ -0,0 +1,14 @@
+import { Component } from '@angular/core';
+
+import { SkyLiveAnnouncerService } from '../live-announcer.service';
+
+@Component({
+  templateUrl: `./live-announcer.component.fixture.html`,
+})
+export class SkyLiveAnnouncerFixtureComponent {
+  constructor(public liveAnnouncerSvc: SkyLiveAnnouncerService) {}
```

Comment:
This can use `inject()`. Also, does it need to be public?

Reply:
Done